app/components/shared/qualifications_component.rb

```diff
@@ -6,13 +6,27 @@ class QualificationsComponent < ViewComponent::Base
 
   alias show_hesa_codes? show_hesa_codes
 
-  def initialize(application_form:, application_choice_state: nil, show_hesa_codes: false)
+  def initialize(application_form:, application_choice: nil, show_hesa_codes: false)
     @application_form = application_form
-    @application_choice_state = application_choice_state
+    @application_choice = application_choice
+    @application_choice_state = application_choice&.status
     @show_hesa_codes = show_hesa_codes
   end
 
   def editable?
     application_form.editable? && current_namespace == 'support_interface'
   end
+
+  def render_degrees?
+    support_interface? ||
+      (provider_interface? && @application_choice.present? && !@application_choice.teacher_degree_apprenticeship?)
+  end
+
+  def support_interface?
+    current_namespace == 'support_interface'
+  end
+
+  def provider_interface?
+    current_namespace == 'provider_interface'
+  end
```

Comment:
Lets just create a new component, one in each interface

Reply:
Sounds good. Done

Reply:
Sounds good. Done